src/ops/ops.go

```diff
@@ -124,6 +125,61 @@ func (o *ops) SystemctlAction(action string, args ...string) error {
 	return errors.Wrapf(err, "Failed executing systemctl %s %s", action, args)
 }
 
+func (o *ops) WriteImageToLocalDevice(liveLogger io.Writer, ignitionPath string, rhelCoreOSImage string) error {
```

Comment:
Does any test provide coverage of this function?

I can see a code path in here that would conditionally execute according to the outcome of some the the calls to ExecPrivilegeCommand

I also think this is quite mockable/testable and there are pre-existing examples of how to approach this.

Reply:
I didn't see much value since there's no actual logic here and every call would be mocked.

I could pull out the ostree output parsing bit since that's the only bit worth testing, otherwise it doesn't feel worth it to me.

Reply:
I think the parsing part should be pulled out so that we can validate the parsing